### DISCUSSION TITLE
单词拼写错误

### DISCUSSION THREAD

**Discussion by afishpapa** [walkor/Workerman#854] [General]
学习框架的时候发先现  \vendor\workerman\workerman\Worker.php  文件  5.0.0版本  单词拼写错误

Line1559       static::log("Warning: User {$this->user} not exsits");
Line1667       static::log("Warning: Group {$this->group} not exsits");

exsits  => exists

**Comment by afishpapa** [comment]
Line 1169            case \SIGTSTP;  //这个分号是不是应该是冒号

**Comment by walkor** [comment]
已经修正，感谢反馈
